vitest.build.mjs

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { defineConfig } from 'vitest/config';
+import process from 'node:process';
+
+export default defineConfig({
+  test: {
+    include: ['./src/build/**/*.test.ts'],
```

Comment:
If I understand correctly, in the old configuration the tests in `src/shared` would be covered by both the browser configuration (`jest.browser.config.js`) and the build configuration (`jest.build.config.js`), wheras after thees changes, they will only be covered by the browser configuration (`vitest.browser.mjs`) and not by the build configuration (`vitest.build.mjs`), because of this include path. Is this intentional?

Reply:
Not intentional, thanks for finding. Fixed now